packages/common/src/types/savedCharts.ts

```diff
@@ -163,7 +163,7 @@ export type Series = {
     };
     name?: string;
     color?: string;
-    yAxisIndex?: number;
```

Comment:
The problem with making this type of change ( optional to required ) is that we aren't migrating the existing chart configs.

Reply:
I seeee, so we should change this back to optional and then address the possible scenarios on the FE only... thanks!
Will create a chart (without the default value) then bring back this code and see how it behaves...